src/data/CommandTree1.15.ts

```diff
@@ -1439,7 +1439,13 @@ export const CommandTree: ICommandTree = {
                     children: {
                         pos: {
                             parser: new VectorArgumentParser(3),
-                            executable: true
+                            executable: true,
+                            children: {
+                                angle: {
+                                    parser: new VectorArgumentParser(1, 'float', false),
+                                    executable: true
+                                }
+                            }
```

Comment:
```suggestion
                            executable: true
```

The `[<angle>]` parameter doesn't exist in 1.15